swatch-product-configuration/src/main/resources/subscription_configs/OpenShift/rhacm.yaml

```diff
@@ -0,0 +1,35 @@
+---
+platform: OpenShift
+
+id: rhacm
+
+variants:
+  - tag: rhacm
+    roles:
+      - moa-hostedcontrolplane
+      - moa
```

Comment:
These roles aren't relevant to RHACM and shouldn't be included in its config.

Reply:
Done. removed